man/systemd-escape.xml

```diff
@@ -159,6 +160,10 @@ tmp-waldi-foobar.mount</programlisting>
     <para>To generate instance names of three strings:</para>
     <programlisting>$ systemd-escape --template=systemd-nspawn@.service 'My Container 1' 'containerb' 'container/III'
 systemd-nspawn@My\x20Container\x201.service systemd-nspawn@containerb.service systemd-nspawn@container-III.service</programlisting>
+
+    <para>To extract the instance part of an instantiated unit:</para>
+    <programlisting>$ systemd-escape -u --template=systemd-nspawn@.service 'systemd-nspawn@My\x20Container\x201.service'
```

Comment:
Hm, I don't think it makes sense to require the template to be specified twice. Why not add a new `--instance` switch, that would take a templated name and give you back the instance? Then `--instance --unescape` should give the unescaped instance name.